.live-plugins/event/plugin.kts

```diff
@@ -0,0 +1,76 @@
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.codeInspection.ProblemHighlightType
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElementVisitor
+import liveplugin.registerInspection
+import org.jetbrains.kotlin.idea.base.utils.fqname.fqName
+import org.jetbrains.kotlin.idea.codeinsight.api.classic.inspections.AbstractKotlinInspection
+import org.jetbrains.kotlin.idea.util.AnnotationModificationHelper
+import org.jetbrains.kotlin.name.FqName
+import org.jetbrains.kotlin.nj2k.postProcessing.type
+import org.jetbrains.kotlin.psi.KtNamedFunction
+import org.jetbrains.kotlin.psi.KtVisitorVoid
+import org.jetbrains.kotlin.psi.psiUtil.isPublic
+import org.jetbrains.kotlin.types.typeUtil.supertypes
+
+// depends-on-plugin org.jetbrains.kotlin
+
+val skyhanniEvent = "at.hannibal2.skyhanni.api.event.SkyHanniEvent"
+val handleEvent = "HandleEvent"
+
+registerInspection(HandleEventInspectionKotlin())
+
+class HandleEventInspectionKotlin : AbstractKotlinInspection() {
+    override fun buildVisitor(holder: ProblemsHolder, isOnTheFly: Boolean): PsiElementVisitor {
+
+        val visitor = object : KtVisitorVoid() {
+            override fun visitNamedFunction(function: KtNamedFunction) {
+                val hasEventAnnotation = function.annotationEntries.any { it.shortName!!.asString() == handleEvent }
+                val isEvent = function.valueParameters.firstOrNull()?.type()?.supertypes()
+                    ?.any { it.fqName?.asString() == skyhanniEvent } ?: false
+
+                if (isEvent && !hasEventAnnotation && function.valueParameters.size == 1 && function.isPublic) {
```

Comment:
idk how but this is a issue
![image](https://github.com/hannibal002/SkyHanni/assets/85900443/912967a3-8929-4d00-a1d8-670f3902d490)

Reply:
this is related to the other plugin not the one added in this one, and its not really wrong but ill push a commit that kind of disables this warning for now until #1983 gets merged

Reply:
cal fixed the issue in beta